Interfaces/en.lproj/DBView.xib

```diff
@@ -3076,9 +3075,36 @@ Gw
                             <outlet property="delegate" destination="68" id="FTJ-Fb-vbI"/>
                         </connections>
                     </textField>
+                    <textField verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="7VL-7D-xfr">
+                        <rect key="frame" x="15" y="146" width="226" height="14"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
+                        <textFieldCell key="cell" lineBreakMode="truncatingMiddle" sendsActionOnEndEditing="YES" alignment="left" title="Duplicate table to different database" id="vT5-w2-d4Q">
```

Comment:
Perhaps instead of "Duplicate table to different database" it could be something like "Destination Database"? To make it a little clearer what the dropdown is doing